pkgs/development/python-modules/rtoml/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, setuptools-rust
+, rustc
+, cargo
+, rustPlatform
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "rtoml";
+  version = "0.6.1";
+  # setuptools-rust requires > 2.7
+  disabled = pythonOlder "3.6";
```

Upstream says[ >= 3.7](https://github.com/samuelcolvin/rtoml/blob/main/setup.py#L30).

```suggestion
  disabled = pythonOlder "3.7";
```